packages/flutter_tools/test/commands.shard/hermetic/attach_test.dart

```diff
@@ -438,7 +564,16 @@ void main() {
     });
 
     testUsingContext('exits when multiple devices connected', () async {
-      final AttachCommand command = AttachCommand();
+      final AttachCommand command = AttachCommand(
```

Comment:
Oh btw, are we still relying on all the overrides here? I would have hoped we could use `testWithoutContext` after this refactor.

Reply:
I just tried using `testWithoutContext` but it seems like the attach command and the `BufferLogger` used in this tests still needs a context

Reply:
Annoyingly `FlutterCommand` depends on globals :(